diff --git a/node_modules/bakong-khqr/src/helper/crc16.js b/node_modules/bakong-khqr/src/helper/crc16.js
index 4feb94a..9ccb928 100644
--- a/node_modules/bakong-khqr/src/helper/crc16.js
+++ b/node_modules/bakong-khqr/src/helper/crc16.js
@@ -42,7 +42,7 @@ function crc16(s) {
     s = Buffer.from(s);
 
     for (i = 0; i < s.length; i++) {
-        c = s[i];
+        var c = s[i];
         j = (c ^ (crc >> 8)) & 0xFF;
         crc = crcTable[j] ^ (crc << 8);
     }
